models/vista2d/configs/inference_trt.json

```diff
@@ -0,0 +1,10 @@
+{
+    "imports": [
+        "$import numpy",
+        "$from monai.networks import trt_compile"
+    ],
+    "trt_args": {
```

Comment:
Thanks @borisfom for the updates.
Based on this PR, I added vista2d and vista3d test cases in your MONAI core PR: https://github.com/Project-MONAI/MONAI/pull/7990
It helps to check the accuracy, and then in model zoo side, we only need to add benchmarks for speed. I will help to do that and push commits into this PR